tests/test_gossip.py

```diff
@@ -1036,10 +1036,10 @@ def test_gossip_addresses(node_factory, bitcoind):
     nodes = l2.rpc.listnodes(l1.info['id'])['nodes']
     assert len(nodes) == 1 and nodes[0]['addresses'] == [
         {'type': 'ipv4', 'address': '127.0.0.1', 'port': 2},
-        {'type': 'ipv4', 'address': '127.0.0.1', 'port': 9735},
+        {'type': 'ipv4', 'address': '127.0.0.1', 'port': 19846},
         {'type': 'ipv6', 'address': '::', 'port': 3},
-        {'type': 'ipv6', 'address': '::', 'port': 9735},
-        {'type': 'torv3', 'address': 'vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion', 'port': 9735},
+        {'type': 'ipv6', 'address': '::', 'port': 19846},
+        {'type': 'torv3', 'address': 'vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion', 'port': 19846},
```

Comment:
Here we need to check the network, in fact, some tests are failing because are running on elements blockchain.

Maybe we need a property before the assert (line 1037) that looks like the following

`port = 19846 if ln.info["network"] is "regtest" else 20735`

I would love to have it at the beginning of the file a list of constant like `LIQUID_PORT = xXX` to make the code more readable

Reply:
Instead of defining the ports for different networks (which would need to be manually changed every time something happens to that code. Yes, I have seen QA departments doing exactly this (rewriting the same code from C to Python for the tests), I think we are understaffed to hurt ourselves this way with no actual reason. The test is still testing all the real things, it just does not care about what default port is chosen when not specified.